libs/libm/newlib/include/math.h

```diff
@@ -0,0 +1,775 @@
+/****************************************************************************
+ * libs/libm/newlib/include/math.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __LIBS_LIBM_NEWLIB_INCLUDE_MATH_H
+#define __LIBS_LIBM_NEWLIB_INCLUDE_MATH_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include <stdint.h>
+
+#include <sys/features.h>
+#include <machine/ieeefp.h>
+#include <_newlib_version.h>
```

Comment:
let's remove the non standard header file: sys/feature.h _newlib_version.h and sys/cdefs.h